test/cfg/gtk.c

```diff
@@ -302,6 +302,7 @@ void g_new0_test()
         int b;
     };
     // valid
+    // cppcheck-suppress valueFlowBailoutIncompleteVar
```

Comment:
This looks like a false positive. Will file a ticket if necessary after this has been merged,